### PR TITLE
refactor:landing-page

### PR DESCRIPTION
**Pull request by gahcalistro**
# Refactor: visual adjustments and removed unnecessary imports/elements/components

## Refactor Landing page
**Just for reference: images on the left are the original and on the
right are with modifications*

1. resized goToTop button and designed an smooth animation
go to top button

2. changed confetti element to other animation using canvas with fade-out effect
now, useEffect and useState is not needed anymore in this case
confetti animation

4. changed icons and highlighted words to make the clickable card intuitive
cards

*[Icons](https://www.svgrepo.com/collection/scarlab-solid-oval-interface-icons/) are from svgRepo and are free to use. 

5. `Hero.tsx` was duplicated 
cards

6. reorganized files to better understand its purposes
cards

7. `loadingContext`
Exported a function calling useContext to avoid import both useContext and LoadingContext unnecessarily. This will allow to import just the data from the context. 
Usage: 
```
import { useLoadingContext } from "contexts/LoadingContext"

const { isLoading } = useLoadingContext()
```

8. removed imports that were not being used

### CODE REVIEW

**Comment by ocodista** [comment]
@gahcalistro Thank you for your contributions!

File: apps/web/app/utils/confetti.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      })
+    )
+    confetti(
+      Object.assign({}, defaults, {
+        particleCount,
+        origin: { x: randomInRange(0.7, 0.9), y: Math.random() - 0.2 },
+      })
+    )
+  }, 250)
+}
```
Comment (ocodista) [inline apps/web/app/utils/confetti.ts:34]: Hey, I liked the structural changes but I think that poping them from random places on the screen it's too agressive.

Could you please refactor this one to drop "fireworks" from the top?

**Review by ocodista — CHANGES_REQUESTED** [review]
Other than the confetti agressiveness, I'll approve the other ones.

Thank you!

**Comment by gahcalistro** [comment]
I've made other changes

1. horizontally centralized and responsively resized images


2. remanaged space in cards



3. reduced size between section description and card.
also increased line heigth to be better to read


4. I know that is just a slight detail but I aligned theses dotted lines


5. I fixed section titles
Smartphones like Galaxy Fold would be affected with this bug.

I'm trying to fix this bug in `HowItWorks" numbered list as well

File: apps/web/app/utils/confetti.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import confetti from 'canvas-confetti'
+
+export default function fireworks() {
+  const duration = 8000 // 8 seconds
```
Comment (ocodista) [inline apps/web/app/utils/confetti.ts:4]:
```suggestion
  const duration = 6_500
```

File: apps/web/app/utils/confetti.ts (hunk 1 above)
Comment (ocodista) [reply apps/web/app/utils/confetti.ts]: I made a commit to update the duration from 8s to 6.5s, other than that it looks great.

Thank you!

**Comment by gahcalistro** [comment]
I was checking lighthouse to see any changes in best practices and there is an error in console.log about "plausible.io". Do you know what is this about? I don't know how to fix it.

**Comment by ocodista** [comment]
> I was checking lighthouse to see any changes in best practices and there is an error in console.log about "plausible.io". Do you know what is this about? I don't know how to fix it. 

I think there's no problem as it doesn't happen on the production.

It is the analytics tool we use, but as you're not running from the domain, it may throw some errors

**Review by ocodista — APPROVED** [review]

**Pull request merged**